src/main/java/frc/robot/commands/drivetrain/MoveDistance.java

```diff
@@ -54,6 +56,7 @@ public void execute() {
   @Override
   public void end(boolean interrupted) {
     drivetrain.stopMotor();
+    timeoutCounter.increaseTimeoutCount();
```

Comment:
This should not be in end(), this should be conditionally triggered in isFinished

Reply:
I have made these changes.